motion_control/drive_controller/include/drive_controller/DifferentialDriveController.hpp

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "etl/math_constants.h"
+
+#include "cogip_defs/Polar.hpp"
+
+#include "utils.hpp"
+
+#include "DriveControllerInterface.hpp"
+#include "DifferentialDriveControllerParameters.hpp"
+#include "motor/MotorInterface.hpp"
+
+namespace cogip {
+
+namespace drive_controller {
+
+class DifferentialDriveController: public DriveControllerInterface {
+public:
+    DifferentialDriveController(DifferentialDriveControllerParameters &parameters,
+                                cogip::motor::MotorInterface &left_motor, 
+                                cogip::motor::MotorInterface &right_motor) :
+                                parameters_(parameters), left_motor_(left_motor), right_motor_(right_motor)
+    {
+    }
+
+    /// @brief Set polar velocity command
+    /// @param command polar velocity command reference
+    /// @return  0 on success, negative on error
+    int set_polar_velocity(cogip_defs::Polar &command) override
+    {
+        // Compute wheel speed in mm/period from polar speed
+        const double left_wheel_speed_mm_period = command.distance() - (command.angle() * parameters_.track_width_mm() / 2);
```

Comment:
prefer `mm_per_period` than `mm_period`, more explicit.
same for other variables

Reply:
Fixed.

Reply:
Nop, you did not apply to all variables, I see `left_wheel_speed_mm_s` for example.

Reply:
Done